File: app/src/main/java/com/example/annada/eventfinder/model/categoryapi/CategoryApiCategory.java
package com.example.annada.eventfinder.model.categoryapi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by : annada
 * Date : 22/10/2017.
 */

public class CategoryApiCategory {
    @SerializedName("categories")
    @Expose
    private List<Category_> categories = null;

    public List<Category_> getCategories() {
        return categories;
    }

    public void setCategories(List<Category_> categories) {
        this.categories = categories;
    }
}

File: app/src/main/java/com/example/annada/eventfinder/model/categoryapi/CategoryApiCategory.kt
package com.example.annada.eventfinder.model.categoryapi

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by : annada
 * Date : 22/10/2017.
 */

class CategoryApiCategory {
    @SerializedName("categories")
    @Expose
    var categories: List<Category_>? = null
}
